src/rpc/util.cpp

```diff
@@ -272,11 +272,12 @@ UniValue DescribeAddress(const CTxDestination& dest)
 
 unsigned int ParseConfirmTarget(const UniValue& value, unsigned int max_target)
 {
-    int target = value.get_int();
-    if (target < 1 || (unsigned int)target > max_target) {
-        throw JSONRPCError(RPC_INVALID_PARAMETER, strprintf("Invalid conf_target, must be between %u - %u", 1, max_target));
+    const int target{value.get_int()};
+    const unsigned int unsigned_target{static_cast<unsigned int>(target)};
+    if (target < 1 || unsigned_target > max_target) {
```

Comment:
Would this not fail if the value of the input was between `MAX_INT` if `max_target` were between `MAX_INT` and `MAX_UNSIGNED_INT`?

Reply:
I assume this is safe, because `max_target` is 1000 blocks, but I did spend a few minutes staring at this.